examples/myrobot/myrobot.py

```diff
@@ -3,16 +3,14 @@
     - simulation of differential robot
 """
 import math
-from threading import Thread
 
+from osgar.node import Node
 from osgar.bus import BusShutdownException
 
 
-class MyTimer(Thread):
+class MyTimer(Node):
     def __init__(self, config, bus):
-        Thread.__init__(self)
-        self.setDaemon(True)
-        self.bus = bus
+        Node.__init__(self, config, bus)
```

Comment:
```suggestion
        super().__init__(config, bus)
```

Reply:
this suggestion should be already applied ... this is strange, but I will double check it.

Reply:
I see it is in `MyTimer` ... which I would at the end move to OSGAR one day. Thanks.